images/rabbitmq/Dockerfile

```diff
@@ -3,9 +3,26 @@ FROM rabbitmq:3-management
 ARG LAGOON_VERSION
 ENV LAGOON_VERSION=$LAGOON_VERSION
 
-COPY rabbitmq_delayed_message_exchange-3.7.0.ez /plugins
+ENV RABBITMQ_DEFAULT_USER=guest \
```

Comment:
can you check that the quotes are consistent. some of the env vars don't have some quotes, some singleticks and some double ticks. It would be nice when they are consistent

Reply:
I'm choosing single ticks for all variables because better protection for special characters like `/` or `^`